actors/builtin/miner/miner_actor.go

```diff
@@ -1512,6 +1515,7 @@ func (a Actor) WithdrawBalance(rt Runtime, params *WithdrawBalanceParams) *adt.E
 
 	currBalance := rt.CurrentBalance()
 	amountWithdrawn := big.Min(st.GetAvailableBalance(currBalance), params.AmountRequested)
+	Assert(amountWithdrawn.GreaterThanEqual(big.Zero()))
```

Comment:
The comment at `verifyPledgeMeetsInitialRequirements` above is a bit misleading now. Also issue #537 will be redundant with this PR.

I see how this works. I think it would be a bit simpler to remove the `verifyPledgeMeetsInitialRequirements` and just explicitly abort (rather than assert) if available balance is <= zero (alt: amountWithdrawn == 0) here.

Reply:
I'm confused, `verifyPledgeMeetsInitialRequirements` does just explicitly abort if available balance is <= 0.

Reply:
Are you saying remove the abort from the verify helper?